File: python_ToSXs/python/08-mang/118.py
arrInt = [1, 2, 3]

result = "Tăng dần"

for i in range(0, len(arrInt) - 1):
    if arrInt[i] >= arrInt[i + 1]:
        result = "Không tăng dần"
        break

print(result)

File: cpp_e9zMZ/cpp/08-mang/118.cpp
#include <iostream>
using namespace std;

int main() {
    int arrInt[] = {-2, -1};
    int length = sizeof(arrInt) / sizeof(int);

    string result = "Tăng dần";
    for (int i = 0; i < length - 1; i++) {
        if (arrInt[i] >= arrInt[i + 1]) {
            result = "Không tăng dần";
            break;
        }
    }

    cout << result;
    return 0;
}